core/src/main/java/hudson/model/Job.java

```diff
@@ -1509,4 +1511,18 @@ public BuildTimelineWidget getTimeline() {
     }
 
     private final static HexStringConfidentialKey SERVER_COOKIE = new HexStringConfidentialKey(Job.class,"serverCookie",16);
+    
+    /**
+     * This handles the situation when Parameterized build checkbox is checked 
+     * but no parameters are selected. User will be redirected to an error page
+     * with proper error message.
+     * @param jsonProperties
```

Comment:
Just for the future. Please avoid using empty Javadoc definitions. Default Javadoc linting behavior in JDK8 considers it as an error